pyro/distributions/empirical.py

```diff
@@ -11,12 +14,21 @@
 @copy_docs_from(TorchDistribution)
 class Empirical(TorchDistribution):
     r"""
-    Empirical distribution associated with the sampled data.
+    Empirical distribution associated with the sampled data. Note that the shape
+    requirement for `log_weights` is that its leftmost shape must match that of
```

Comment:
nit: its shape must match the leftmost shape of

Reply:
Good catch, will fix.